recipes/psipred/perlwrapper.patch

```diff
@@ -0,0 +1,179 @@
+--- runpsipred.pl	1969-12-31 16:00:00.000000000 -0800
++++ runpsipred.stefan	2017-02-22 12:03:16.986281400 -0800
+@@ -0,0 +1,176 @@
++#!/usr/bin/perl
++
++# This is a wrapper script to run PSIPRED
++# written by Stefan Janssen, Feb 2017
++
++use strict;
++use warnings;
++use Getopt::Long;
++use File::Basename;
++use Data::Dumper;
++use File::Spec;
++
++my $execdir = "/opt/anaconda1anaconda2anaconda3/bin/";
++my $datadir = "/opt/anaconda1anaconda2anaconda3/share/psipred_4.01/data/";
++
++my $inputfile = undef;
++my $dbname = undef;
++my $ncbidir = "";
++my $mtx = 0;
++
++&get_arguments();
++
++# Generate a "unique" temporary filename root
++my $hostid = qx(uname -n); chomp $hostid;
++my $tmproot = ($ENV{TMPDIR} ? exists $ENV{TMPDIR} : '').'psitmp'.$$.$hostid;
++qx(mkdir -p $tmproot && cp -f $inputfile $tmproot/input); #($? != 0)
++
++if ($mtx != 1) {
```

Comment:
we are lazy in Perl. We assume that the default behaviour is running blast and only if the user read the doc and knows what he does he would set mtx to 1 to change default behaviour. Thus, I don't want to bother the user with incorrectly set values for mtx.